lsc20-generic-extraction.json

```diff
@@ -0,0 +1,49 @@
+{
+  "type": "IMAGE_SEQUENCE",
+  "input":{
+    "path": "/home/ubuntu/vitrivr/data/lsc2020-images/",
+    "depth": 2,
+    "id": {
+      "name": "FileNameObjectIdGenerator",
+      "properties": {}
+    }
+  },
+  "extractors":[
+    {"name": "AverageColor"},
+    {"name": "AverageColorARP44"},
+    {"name": "AverageColorARP44Normalized"},
+    {"name": "AverageColorGrid8"},
+    {"name": "AverageColorGrid8Normalized"},
+    {"name": "AverageColorRaster"},
+    {"name": "AverageFuzzyHist"},
+    {"name": "CLD"},
+    {"name": "CLDNormalized"},
+    {"name": "ConceptMasksAde20k"},
+    {"name": "DominantEdgeGrid8"},
+    {"name": "DominantEdgeGrid16"},
+    {"name": "EHD"},
+    {"name": "HueHistogram"},
+    {"name": "MedianColor"},
+    {"name": "MedianColorGrid8"},
+    {"name": "MedianFuzzyHist"},
+    {"name": "HOGMirflickr25K256"},
+    {"name": "HOGMirflickr25K512"},
+    {"name": "SubDivMotionHistogram3"},
+    {"name": "SubDivMotionHistogram5"},
+    {"name": "SubDivMotionHistogramBackground3"},
+    {"name": "SubDivMotionHistogramBackground5"}
+  ],
+  "exporters":[
+    {
+      "name": "ShotThumbnailsExporter",
+      "properties": {
+        "destination":"./lsc2020-thumbnails-all"
+      }
+    }
+  ],
+  "database":{
+    "selector": "JSON",
+    "writer": "JSON",
+    "host": "./json-extraction"
+  }
+}

```

Comment:
Can't this be deleted as well?

Reply:
Yeah, that one slipped as well, it will get removed too. Anything LSC related is archived in the [lsc-tag](https://github.com/vitrivr/cineast/releases/tag/lsc2020)